### PR TITLE
Add Screenshot preview screen, routing mechanism(WIP)

### PR DESCRIPTION
**Pull request by kabiir**

### CODE REVIEW

File: android/key.properties [hunk 1]
```diff
@@ -0,0 +1,4 @@
+storePassword=android
+keyPassword=android
```
Comment (kabiir) [inline android/key.properties]: Just cause it's a debug key. Hence the password

File: lib/main.dart [hunk 2]
```diff
@@ -1,73 +1,28 @@
...
 
+void main() {
+  runApp(new TriveApp());
 }
 
-
-class MyHomePage extends StatefulWidget {
-  MyHomePage({Key key, this.title}) : super(key: key);
-  final String title;
+class TriveApp extends StatefulWidget {
```
Comment (kabiir) [inline lib/main.dart]: Added back the TriveApp WidgetClass for easier testing

File: lib/main.dart [hunk 3]
```diff
@@ -1,73 +1,28 @@
...
-          ],
-        ),
-      ),
-      floatingActionButton: new FloatingActionButton(
-        onPressed: getImage,
-        tooltip: 'Pick Image',
-        child: new Icon(Icons.add_a_photo),
-      ),
+    return new MaterialApp(
+      theme: new ThemeData(
```
Comment (kabiir) [inline lib/main.dart]: Should eventually be moved to its own `.dart` file along with the rest of design schematics once designers provide mockups.

File: pubspec.yaml [hunk 4]
```diff
@@ -5,7 +5,7 @@ dependencies:
   flutter:
     sdk: flutter
 
-  image_picker: any
+  image_picker: "^0.2.1"
```
Comment (kabiir) [inline pubspec.yaml]: We should always specify version numbers to avoid unnecessary hiccups.

**Comment by kabiir** [comment]
I admit it's a lot of stuff in one PR and should be avoided in the future. PRs should be either feature specific or hotfix/bufixes but not both.

**Comment by eukreign** [comment]
Manually merged pull request after applying some changes to it.

**Pull request closed without merging**